docs/source/help-support.rst

```diff
@@ -1,35 +1,49 @@
-=================
+================
 Help and support
-=================
+================
 
-* Get free community support with our `Google group
-  <https://groups.google.com/a/anaconda.com/forum/#!forum/anaconda>`_.
+Free community support
+======================
 
-* Paid `support <https://www.anaconda.com/support/>`_,
-  `training <https://www.anaconda.com/training/>`_ and
-  `consulting <https://www.anaconda.com/consulting/>`_
-  options are available. Our teaching philosophy is that the best
-  way to learn is with hands-on experience addressing real-world
-  problems. Courses are available to individuals online, at
-  numerous sites, or in-house at your place of business. We also
-  offer consulting services for the analysis, management and
-  visualization of scientific and business data or optimizing
-  your processing workflows on modern hardware and GPUs.
+Ask and answer questions, discuss ways to use conda,
+request new features, and submit any other comments you might have.
 
-* Support is included in purchases of `Anaconda Enterprise
-  <https://www.anaconda.com/enterprise/>`_.
+.. grid:: 1 2 2 2
```

Comment:
Not sure if it's GH's "display rich diff" glitching out but when I select that option, this part shows up and I think it's supposed to be hidden (see screenshot below)

![Screenshot 2024-04-18 at 12 47 08 PM](https://github.com/conda/conda-docs/assets/28930622/cbdb9d79-31cc-4ef8-8de9-9968a1ea6f6e)

Reply:
I think it's just a glitch. My local build looks like this:

![image](https://github.com/conda/conda-docs/assets/90850002/c59e8394-7e19-405b-9a73-f6a127340c6a)